examples/notebooks/healthcare/beam_post_hl7_messages_to_hcapi.ipynb

```diff
@@ -0,0 +1,504 @@
+      ],
+      "metadata": {
+        "id": "RL1LDp645ogr"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# **Post Hl7v2 messages to Google Cloud Healthcare API HL7v2 store pipeline**\n",
+        "\n",
+        "This example demonstrates how to set up an Apache Beam pipeline that reads a HL7 file from [Google Cloud Storage](https://https://cloud.google.com/storage), and calls the [Google Cloud Healthcare API Hl7v2 store to store Hl7 messages](https://cloud.google.com/healthcare-api/docs/how-tos/hl7v2-messages) to extract information from unstructured data. This application can be used in contexts such as reading raw Hl7 messages, if needed parse them or modify them as per your defined Hl7v2 store configurations and store data into Hl7v2 store.\n",
+        "\n",
+        "An Apache Beam pipeline is a pipeline that reads input data, transforms that data, and writes output data. It consists of PTransforms and PCollections. A PCollection represents a distributed data set that your Beam pipeline operates on. A PTransform represents a data processing operation, or a step, in your pipeline. It takes one or more PCollections as input, performs a processing function that you provide on the elements of that PCollection, and produces zero or more output PCollection objects.\n",
+        "\n",
+        "For details about Apache Beam pipelines, including PTransforms and PCollections, visit the [Beam Programming Guide](https://beam.apache.org/documentation/programming-guide/).\n",
+        "\n",
+        "You'll be able to use this notebook to explore the data in each PCollection."
+      ],
+      "metadata": {
+        "id": "wC9KRrlORwKu"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "***Sample HL7v2 Message***\n",
+        "\n",
+        "The below reference message shows a sample Hl7v2 messages seperated by \\r.\n",
```

Comment:
It would probably be helpful to briefly describe what information this message contains. As someone who isn't familiar with these kinds of messages, I'm not sure how to read it

Reply:
Added the information and details regarding Hl7 message.